src/Factory/ViewFactory.php

```diff
@@ -10,11 +10,14 @@
 use Yiisoft\Router\UrlGeneratorInterface;
 use Yiisoft\View\Theme;
 use Yiisoft\View\WebView;
+use Yiisoft\Widget\WidgetFactory;
 
 class ViewFactory
 {
     public function __invoke(ContainerInterface $container)
     {
+        WidgetFactory::initialize($container);
```

Comment:
I think this can be placed into container configuration (`config/common.php`).

Reply:
how do you imagine it?

Reply:
Like [that](https://github.com/yiisoft/yii-dataview/blob/master/config/tests.php#L30)

Reply:
Where and how will this definition be used? No one needs `WidgetFactory` as a dependency

Reply:
Ok, we can solve it later